gulpfile.js/tasks/build-scripts.js

```diff
@@ -20,7 +20,7 @@ const extensions = ['.js', '.ts', '.tsx'];
  */
 const createVirtualRootEntry = async () => {
   const files = (
-    await glob(`src/{objects,components}/**/*{${extensions.join(',')}}`)
+    await glob(`src/{objects,components}/*/*{${extensions.join(',')}}`)
```

Comment:
Had to change this because it was including `src/components/pagination/demo/mock-pagination-data.js` when it shouldn't have been

Reply:
If I'm reading this right, this is saying we'll only grab JS files from one sublevel inside `objects` and `components`. Is that a safe assumption? Are we sure we'll never need another level or organization?

If the use case you're avoiding is not including the demo data, would it be easier to exclude the `demo` folders?

Reply:
> Are we sure we'll never need another level of organization?

I was trying to make it so that it would only include files directly from the top-level of each component (i.e. `src/components/button/____.js` but not `src/components/button/______/_____.js`). I did not think about the case of potentially nesting components a level deeper (like `src/components/some_group/button/_____.js`). I think if we decide to do that we can then update this glob.

I think we want to exclude any subfolder of components, not just `demo`. If a component had a subfolder in it, I doubt that we would want it to provide all of the exports of all of those files as top-level exports of the pattern library. If we did want to export anything from those, we could opt-in for each one by adding `export {____} from './____'` to a file in the top-level of the component

Reply:
That tracks. And if we did end up needing component groups or subcomponents, we can always revisit this decision at that point.